ctre/basemotorcontroller.py

```diff
@@ -190,12 +194,14 @@ def setInverted(self, invert: bool):
         :param invert:
             Invert state to set.
         """
-        self.invert = invert
-        super().setInverted(invert)
+        self.invert = int(invert)
```

Comment:
I'm not too sure whether depending on the enum variant values, unlike the Java version, is a great idea here. This feels rather magic to me.

Reply:
Tests will catch it if the enum changes, so I'm fine with it.